### PR TITLE
Fixing the broken rendering of subsurface scattering

### PR DESCRIPTION
**Pull request by samcake**
After merging the new Texture Streaming project we ahve discovered a few bugs.
THis pr fixes the rendering of the AO and Subsurface scattering not rendering properly.
THis PR fix the bug for OpenGL 4.5 platforms (PC windows or linux) not for 4.1 backend (mac and intel gpu  pc). WE are working on that next

## TEST PLAN
- Using Matthew or Pricilia avatar model, their skin should show correctly with this PR and be corrupted with current master

- Turning AO effect own should work in this PR and not in current Master

### CODE REVIEW

File: libraries/render-utils/src/AmbientOcclusionEffect.cpp [hunk 1]
```diff
@@ -74,11 +74,12 @@ void AmbientOcclusionFramebuffer::allocate() {
     auto width = _frameSize.x;
     auto height = _frameSize.y;
     
-    _occlusionTexture = gpu::TexturePointer(gpu::Texture::create2D(gpu::Element::COLOR_RGBA_32, width, height, gpu::Sampler(gpu::Sampler::FILTER_MIN_MAG_LINEAR_MIP_POINT)));
+ //   _occlusionTexture = gpu::TexturePointer(gpu::Texture::create2D(gpu::Element::COLOR_RGBA_32, width, height, gpu::Sampler(gpu::Sampler::FILTER_MIN_MAG_LINEAR_MIP_POINT)));
```
Comment (ZappoMan) [inline libraries/render-utils/src/AmbientOcclusionEffect.cpp]: dead code?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 618344fe98aa712871c1513a92c7a6e571fef9be available at: https://gist.github.com/a0ee604745357455b164e4d6aacbc7d7<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9992-618344f.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9992-618344f.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 022e91b35dedfa3420e8b094909a4ad4bdd5c2c8 available at: https://gist.github.com/a0ee604745357455b164e4d6aacbc7d7<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9992-022e91b.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9992-022e91b.dmg

File: libraries/gpu-gl/src/gpu/gl/GLTexture.cpp [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 #include "GLTexture.h"
 
-#include <NumericalConstants.h>
+//#include <NumericalConstants.h>
```
Comment (ZappoMan) [inline libraries/gpu-gl/src/gpu/gl/GLTexture.cpp]: dead code?

**Comment by hifi-gustavo** [comment]
Build console output for commit: a0488cfb912d03ff1cb63544bc98c9db1254dcf4 available at: https://gist.github.com/a0ee604745357455b164e4d6aacbc7d7<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9992-a0488cf.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9992-a0488cf.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 13b3809fe325e031b48c62542d2e6be5a7d5d305 available at: https://gist.github.com/a0ee604745357455b164e4d6aacbc7d7<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9992-13b3809.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9992-13b3809.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ad5c78349a02a2e1835648699c7e164498d9a1c3 available at: https://gist.github.com/a0ee604745357455b164e4d6aacbc7d7<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9992-ad5c783.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9992-ad5c783.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 25d39d2430e26d1a74e05427fcb4ee4de75677db available at: https://gist.github.com/a0ee604745357455b164e4d6aacbc7d7<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9992-25d39d2.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9992-25d39d2.dmg

**Pull request merged**